spec/debugging_issue_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+class DoerOfSomething
+  def self.do_something
+    'something done by the DoerOfSomething'
+  end
+end
+
+class DoSomething < GraphQL::Schema::RelayClassicMutation
+  field :what_it_did, String
+
+  def authorized?
+    true
+  end
+
+  def resolve
+    puts 'Resolving doSomething mutation'
+
+    result = { what_it_did: DoerOfSomething.do_something }
+
+    puts "Something was done, we'll now return the value!"
+
+    result
+  end
+end
+
+class RootMutation < GraphQL::Schema::Object
+  field :do_something, mutation: DoSomething
+end
+
+class TestSchema < GraphQL::Schema
+  mutation RootMutation
+
+  use(GraphQL::Dataloader)
```

Comment:
The bug only happens when using the dataloader. Seems related to the usage of fibers.

Note that the dataloader is not being used anywhere, we just need to plug it in here for the issue to happen.